<launch>

    <node pkg="joy" type="joy_node" name="joy_node" output="screen"/>

    <node pkg="logi_teleop" type="joy_control" name="joy_control" output="screen">
        <param name="MaxLinear" value="0.5"/>
        <param name="MaxAngular" value="0.3"/>
        <param name="forward" value="1"/>
        <param name="turn" value="2"/>
        <param name="add_vel" value="1"/>
        <param name="dec_vel" value="3"/>
        <param name="add_ang" value="0"/>
        <param name="dec_ang" value="2"/>
        <param name="reset" value="9"/>
        <remap from="cmd_vel" to="exp_vels"/>
    </node>

</launch>